vital/algo/estimate_similarity_transform.cxx

```diff
@@ -82,46 +86,49 @@ namespace
  * \param to_pts        vector in which to store \c to points that have
  *                      a corresponding \c from point.
  */
-template< typename M,
-          vector_3d (M::value_type::second_type::element_type::*afunc)() const >
-void map_to_pts(M const& from_map, M const& to_map,
-                std::vector<vector_3d> &from_pts, std::vector<vector_3d> &to_pts)
+template < typename M,
+           vector_3d( M::value_type::second_type::element_type::* afunc )( ) const >
+void
+map_to_pts( M const& from_map, M const& to_map,
+            std::vector< vector_3d >& from_pts,
+            std::vector< vector_3d >& to_pts )
 {
   typename M::const_iterator from_it = from_map.begin(),
                              to_it   = to_map.begin();
   // STL map structure's stored data is ordered (binary search tree impl
   // O(from.size + to.size)
-  while (from_it != from_map.end() && to_it != to_map.end())
+  while( from_it != from_map.end() && to_it != to_map.end() )
   {
     // increment the lesser of the two when the frame IDs don't match
-    if (from_it->first > to_it->first)
+    if( from_it->first > to_it->first )
     {
       ++to_it;
     }
-    else if (from_it->first < to_it->first)
+    else if( from_it->first < to_it->first )
     {
       ++from_it;
     }
     else // equal
     {
-      from_pts.push_back( ((*from_it->second).*afunc)() );
-      to_pts.push_back( ((*to_it->second).*afunc)() );
-      ++from_it; ++to_it;
+      from_pts.push_back( ( ( *from_it->second ).*afunc )() );
+      to_pts.push_back( ( ( *to_it->second ).*afunc )() );
+      ++from_it;
+      ++to_it;
     }
   }
 }
 
-}
+} // namespace
```

Comment:
```suggestion
} // namespace <anonymous>
```
(Yeah, not implemented in Uncrustify yet...)

Reply:
Fixed.